README.md

```diff
@@ -299,7 +299,7 @@ In order to achieve greater coverage and encourage more people to contribute to
         <tr>
             <td><a href="https://en.wikipedia.org/wiki/Linear_search">Linear Search (Iterative)</a></td>
             <td> <!-- C -->
-                <a href="./src/c/BuscaSequencial.c">
+                <a href="./src/c/SequentialSearch.c">
```

Comment:
What do you think about renaming it to `LinearSearch`? (to match the section name)

Reply:
Yes, that makes more sense and the read is better

Reply:
solved in the lasts commits